### PR TITLE
Failed Environment UI View

### PR DESCRIPTION
**Pull request by esterlinej**
### FEATURE
Now that ChartErrors are being persisted to the database for environment failures, we need to implement a UI view that displays this information in a friendly way to users for debugging purposes.

### UPDATES
- Utilized prior static environment failure report template as part of this implementation, updated format & layout
- Dynamically inserts status & button (link) to `failure_report` failure UI view with breadcrumb
- Cleanup old template and references

### CODE REVIEW

**Comment by esterlinej** [comment]
Screen Shot 2020-12-04 at 12 23 55 PM
Screen Shot 2020-12-04 at 3 09 55 PM
Screen Shot 2020-12-04 at 3 15 59 PM

File: cmd/server.go [hunk 1]
```diff
@@ -90,7 +89,6 @@ func init() {
 	serverCmd.PersistentFlags().StringVar(&k8sGroupBindingsStr, "k8s-group-bindings", "", "optional k8s RBAC group bindings (comma-separated) for new environment namespaces in GROUP1=CLUSTER_ROLE1,GROUP2=CLUSTER_ROLE2 format (ex: users=edit) (Nitro)")
 	serverCmd.PersistentFlags().StringVar(&k8sSecretsStr, "k8s-secret-injections", "", "optional k8s secret injections (comma-separated) for new environment namespaces in SECRET_NAME=VAULT_ID (Vault path using secrets mapping) format. Secret value in Vault must be a JSON-encoded object with two keys: 'data' (map of string to base64-encoded bytes), 'type' (string). (Nitro)")
 	serverCmd.PersistentFlags().StringVar(&k8sPrivilegedReposStr, "k8s-privileged-repo-whitelist", "dollarshaveclub/acyl", "optional comma-separated whitelist of GitHub repositories whose environment service accounts will be allowed cluster-admin privileges (Nitro)")
-	serverCmd.PersistentFlags().StringVar(&failureTemplatePath, "failure-template-path", "/opt/html/failedenv.html.tmpl", "path to HTML failure report template (if missing, failure reports will be disabled")
```
Comment (bkeroackdsc) [inline cmd/server.go:93]: Has this been removed from the chart as well?

**Review by bkeroackdsc — APPROVED** [review]

File: cmd/server.go (hunk 1 above)
Comment (esterlinej) [reply cmd/server.go:93]: No reference exists for this in the current chart, I checked both local and in the chart repo.

**Comment by wwalexander** [comment]
LGTM

**Pull request merged**